build/release/Makefile

```diff
@@ -19,11 +19,25 @@ include ../makelib/common.mk
 # ====================================================================================
 # Options
 
+# set the shell to bash in case some environments use sh
+SHELL := /bin/bash
+
 CHANNEL ?= master
 ifeq ($(filter master alpha beta stable,$(CHANNEL)),)
 $(error invalid channel $(CHANNEL))
 endif
 
+DOCS_VERSION := $(shell echo $(BRANCH_NAME) | sed -E "s/^release\-([0-9]+)\.([0-9]+)$$/v\1.\2/g")
+DOCS_DIR ?= $(ROOT_DIR)/Documentation
+DOCS_WORK_DIR := $(WORK_DIR)/rook.github.io
+DOCS_VERSION_DIR := $(DOCS_WORK_DIR)/docs/rook/$(DOCS_VERSION)
+
+ifdef GIT_API_TOKEN
+DOCS_GIT_REPO := https://$(GIT_API_TOKEN)@github.com/rook/rook.github.io.git
+else
+DOCS_GIT_REPO := git@github.com:rook/rook.github.io.git
```

Comment:
why is this not https://github.com:rook/rook.github.io.git? I don't use SSH for GitHub for example.